opal-ui/src/router/routes.ts

```diff
@@ -14,6 +14,7 @@ const routes: RouteRecordRaw[] = [
       { path: '/admin/settings', component: () => import('pages/AdminConfigurationPage.vue') },
       { path: '/admin/databases', component: () => import('pages/AdminDatabasesPage.vue') },
       { path: '/admin/profiles', component: () => import('pages/AdminProfilesPage.vue') },
+      { path: '/admin/profile', component: () => import('pages/AdminProfilePage.vue') },
```

Comment:
If it is "my profile", it is not in the admin space, let it at `/profile`.

Reply:
Moved the files to `/profile`